packages/nitro-protocol/test/contracts/MultiAssetHolder/computeTransferEffectsAndInteractions.test.ts

```diff
@@ -12,50 +13,57 @@ const testNitroAdjudicator = (setupContract(
   process.env.TEST_NITRO_ADJUDICATOR_ADDRESS
 ) as unknown) as TESTNitroAdjudicator;
 
-import {AllocationItem} from '../../../src';
-import {computeNewAllocation} from '../../../src/contract/multi-asset-holder';
+import {computeTransferEffectsAndInteractions} from '../../../src/contract/multi-asset-holder';
 
-const randomAllocation = (numAllocationItems: number): AllocationItem[] => {
-  return numAllocationItems > 0
-    ? [...Array(numAllocationItems)].map(() => ({
+const randomAllocations = (numAllocations: number): Allocation[] => {
+  return numAllocations > 0
+    ? [...Array(numAllocations)].map(() => ({
         destination: randomExternalDestination(),
         amount: BigNumber.from(Math.ceil(Math.random() * 10)).toHexString(),
+        metadata: '0x',
+        allocationType: AllocationType.simple,
       }))
     : [];
 };
 
 const heldBefore = BigNumber.from(100).toHexString();
-const allocation = randomAllocation(Math.floor(Math.random() * 20));
+const allocation = randomAllocations(Math.floor(Math.random() * 20));
 const indices = shuffle([...Array(allocation.length).keys()]); // [0, 1, 2, 3,...] but shuffled
+// TODO -- does it make sense to test with indices that don't increase when the chain requires that they do?
 
-describe('AssetHolder._computeNewAllocation', () => {
+describe('MultiAssetHolder.compute_transfer_effects_and_interactions', () => {
   it(`matches on chain method for input \n heldBefore: ${heldBefore}, \n allocation: ${JSON.stringify(
     allocation,
     null,
     2
   )}, \n indices: ${indices}`, async () => {
     // check local function works as expected
-    const locallyComputedNewAllocation = computeNewAllocation(heldBefore, allocation, indices);
+    const locallyComputedNewAllocation = computeTransferEffectsAndInteractions(
+      heldBefore,
+      allocation,
+      indices
+    );
 
-    const result = await testNitroAdjudicator._computeNewAllocation(
+    const result = await testNitroAdjudicator.compute_transfer_effects_and_interactions(
       heldBefore,
       allocation,
```

Comment:
This looks like it might be due to an ethers version mismatch between `nitro-protocol` and `exit-format` 😞 .

Reply:
In newer versions of ethers (since [5.2.0](https://github.com/ethers-io/ethers.js/releases/tag/v5.2.0)), the javascript `bigint` was added to the `BigNumberish` union. The exit format exports types that use that newer and wider union -- which causes an error when we try to pass them into typechain functions that don't expect a `bigint`.

Reply:
@NiloCK could you try linking against this branch https://github.com/statechannels/exit-format/pull/63 and running the tests again?

Reply:
This worked. A clean `yarn`, `yarn prepare` with the linked package runs the test failure free and silences VSCode.